React/CoreModules/React-CoreModules.podspec

```diff
@@ -30,8 +30,9 @@ Pod::Spec.new do |s|
   s.platforms              = { :ios => "9.0", :tvos => "9.2" }
   s.compiler_flags         = folly_compiler_flags + ' -Wno-nullability-completeness'
   s.source                 = source
-  s.source_files           = "**/*.{c,h,m,mm,cpp}"
-  s.header_dir             = "React"
+  s.source_files           = "**/*.{c,m,mm,cpp}"
+  s.header_dir             = "CoreModules"
```

Comment:
I think this one would be a breaking change (more files will be moved here for TurboModule migration), I'll see if I can move it up as a subspec inside React-Core.podspec

Reply:
nvm, I think this is meaningless now that you removed the .h files from the list

Reply:
Yes, thats right.